python/test/t_stiffened_panel.py

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env python
+
+import openturns as ot
+from openturns.testing import assert_almost_equal
+from openturns.usecases import stiffened_panel as stiffened_panel
+
+
+ot.TESTPREAMBLE()
+ot.PlatformInfo.SetNumericalPrecision(20)
+
+print("Test the StiffenedPanel class.")
+sp = stiffened_panel.StiffenedPanel()
+
+print(sp.distribution.getMean())
+# test parameter
+assert sp.dim == 10
+
+# test marginals means
+assert_almost_equal(sp.youngModulus.getMean()[0], 110000000000, 1e-12)
+assert_almost_equal(sp.nu.getMean()[0], 0.375, 1e-12)
+assert_almost_equal(sp.h_c.getMean()[0], 0.03, 1e-12)
+assert_almost_equal(sp.ell.getMean()[0], 0.049, 1e-12)
+assert_almost_equal(sp.f_1.getMean()[0], 0.028, 1e-12)
+assert_almost_equal(sp.f_2.getMean()[0], 0.0066, 1e-12)
+assert_almost_equal(sp.t.getMean()[0], 8.1005e-5, 1e-12)
+assert_almost_equal(sp.a.getMean()[0], 0.61, 1e-12)
+assert_almost_equal(sp.b_0.getMean()[0], 0.045, 1e-12)
+assert_almost_equal(sp.p.getMean()[0], 0.038, 1e-12)
+
+meanInput = sp.distribution.getMean()
+print("meanInput = ", meanInput)
+meanOutput = sp.model(meanInput)
+print("meanOutput = ", meanOutput)
+assert_almost_equal(meanOutput, [197.76], 1.0e-4)
```

Comment:
The input mean is known to machine precision. Therefore, the mean output (to first order) is also known to machine precision. I suggest to get the 17 significant, remove the last two ones and set this as the reference value, with a relative tolerance of 1.e-15.